configure.ac

```diff
@@ -427,6 +427,11 @@ AC_CHECK_SIZEOF(wchar_t)
 
 AC_CHECK_SIZEOF(pid_t)
 
+# Check sizes of atomic types so we can define fixed-width types in OPAL
+AC_CHECK_SIZEOF(atomic_short, [],[[#include <stdatomic.h>]])
+AC_CHECK_SIZEOF(atomic_int,[],[[#include <stdatomic.h>]])
+AC_CHECK_SIZEOF(atomic_long,[],[[#include <stdatomic.h>]])
+AC_CHECK_SIZEOF(atomic_llong,[],[[#include <stdatomic.h>]])
```

Comment:
The code that uses these defines will be added soon. The problem is that C11 didn't define fixed-width atomic type (atomic_uint32_t, etc) so we will have to define them ourselves. The problem is that Intel only implemented half of C11 atomics. They do not support _Atomic. This means we have to define atomic_uint32_t, etc terms of the atomic_int, atomic_long, etc.